ui/partials/perf/testdatachooser.html

```diff
@@ -28,10 +28,10 @@ <h3 class="modal-title">Add test data</h3>
     </div>
   </div>
   <div class="btn-group-vertical ">
-    <button id="unselect-test" ng-click="unselectTest()" type="button" class="btn btn-xs btn-default">
+    <button id="unselect-test" ng-click="unselectTest()" type="button" class="btn btn-xs btn-default" ng-disabled="!testsToAdd.length">
       <span class="glyphicon glyphicon-chevron-left"></span>
     </button>
-    <button id="select-test" ng-click="selectTest()" type="button" class="btn btn-xs btn-default">
+    <button id="select-test" ng-click="selectTest()" type="button" class="btn btn-xs btn-default" ng-disabled="!unselectedTestList.length">
```

Comment:
These seem to look fine to me. I defer to you guys if you want to wrap the attributes/directives for shorter lines.